docs/source/developer-guides/index.rst

```diff
@@ -0,0 +1,9 @@
+Developer Guides
+=======================
+
+The following pages provide information for existing and prospective developers about how to work effectively in the Kokkos team.
+
+.. toctree::
+   :maxdepth: 1
+
+   PRs and Reviews <prs-and-reviews.rst>
```

Comment:
Just a note, if you adopt the below heading change you don't need to rename the link here. It doesn't matter too much though.